src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java

```diff
@@ -874,6 +877,42 @@ public Message createMessage(JSONObject jsonObject, MessageChannel chan, boolean
         final List<MessageEmbed>       embeds      = map(jsonObject, "embeds",      this::createMessageEmbed);
         final List<MessageReaction>    reactions   = map(jsonObject, "reactions",   (obj) -> createMessageReaction(chan, id, obj));
 
+        if (!jsonObject.isNull("activity"))
+        {
+            JSONObject activityData = jsonObject.getJSONObject("activity");
+            Message.ActivityType activityType = Message.ActivityType.fromId(activityData.getInt("type"));
+            final String partyId = activityData.getString("party_id");
+            Message.Application application = null;
+
+            switch (activityType)
+            {
+                case PARTY:
+                    activity = new Message.Activity(activityType, partyId, application);
+                    break;
+                case GAME:
+                    {
```

Comment:
put the scope on the same column as the case
```java
case GAME:
{
    ...
```

Either use scopes on all cases or none